### DISCUSSION TITLE
Can I manage future changes in specific records with this package?

### DISCUSSION THREAD

**Discussion by jeetu-sah** [spatie/laravel-activitylog#891] [Q&A]
Can I manage future changes in specific records with this package? 
  I have a task in which I have a need to store all changes that changes will affect in future from a specific date and time.
   And user can delete and modify that log if the user deletes that specific or selected record. It's mean the changes will be revert according to the last one.
  Same as work in modifying.

**Comment by Gummibeer** [comment]
I have no idea what you want but you should check out event sourcing as it sounds like that would solve your problem.
https://github.com/spatie/laravel-event-sourcing
https://github.com/spatie/laravel-eventsauce
https://eventsauce.io
https://event-sourcing-laravel.com
